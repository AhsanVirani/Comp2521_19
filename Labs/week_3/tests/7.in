10
20
15
30
25
40
35
50
45
60
55
65
