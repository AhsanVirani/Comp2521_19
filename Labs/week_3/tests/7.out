Tree:
10
  \
  20
  / \
 /   \
15   30
     / \
    /   \
   25   40
        / \
       /   \
      35   50
           / \
          /   \
         45   60
              / \
             /   \
            55   65

#nodes = 12,  #leaves = 6
Infix  : 10 15 20 25 30 35 40 45 50 55 60 65 
Prefix : 10 20 15 30 25 40 35 50 45 60 55 65 
Postfix: 15 25 35 45 55 65 60 50 40 30 20 10 
ByLevel: 10 20 15 30 25 40 35 50 45 60 55 65 
